src/mono/System.Private.CoreLib/src/System/Threading/ThreadPool.Browser.Mono.cs

```diff
@@ -98,6 +92,13 @@ internal static void NotifyThreadUnblocked()
 
         internal static object? GetOrCreateThreadLocalCompletionCountObject() => null;
 
+#pragma warning disable IDE0060
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
I'm surprised this attribute would be needed.

Reply:
I just moved the code around but yeah it seems redundant